docs/ref/databases.txt

```diff
@@ -941,6 +941,27 @@ To enable the JSON1 extension you can follow the instruction on
 .. _JSON1 extension: https://www.sqlite.org/json1.html
 .. _the wiki page: https://code.djangoproject.com/wiki/JSON1Extension
 
+.. _sqlite-init-command:
+
+Setting PRAGMA Options
+----------------------
+
+.. versionadded:: 5.1
+
+Pragma options can be set upon connection by using the ``init_command`` option
```

Comment:
I'd add a link to the `https://www.sqlite.org/pragma.html`
```suggestion
:ref:`Pragma options <https://www.sqlite.org/pragma.html>` can be set upon connection by using the ``init_command`` option
```
Remember to re-wrap at 79 chars.

Reply:
I'm a bit new to Sphinx; when I did that exact line I was getting a `WARNING: undefined label` error. I added it where the link reference is defined separately (this seems to be similar to how it is done elsewhere in this file).